### PR TITLE
[orc8r][tools] Clean up combine_swagger

### PR DESCRIPTION
**Pull request by hcgatewood**
## Summary

Note: this PR is actually ~100 lines long -- the length comes from checking-in the full swagger.yml file as a static asset

This PR does a couple things, all related to the combine_swagger command and wintern preparation
- clean up combine_swagger implementation, including reporting merge warnings
- generate full/combined swagger.yml during build.py -g
- generate full/combined swagger.yml as a static asset checked into the codebase (@amarpad)

## Test Plan

- [x] build.py -t
- [x] build.py -g and inspect results
- [x] Added regresion test for combine_swagger

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Comment by 119Vik** [comment]
RestartCWAGTests

**Comment by 119Vik** [comment]
RestartCWAGTests

**Comment by 119Vik** [comment]
RestartCWAGTests

**Comment by 119Vik** [comment]
RestartCWAGTests

File: orc8r/cloud/Makefile [hunk 1]
```diff
@@ -1,8 +1,9 @@
 SHELL := /bin/bash
 
-################
-## Essentials ##
-################
+##########################
+## Essentials variables ##
```
Comment (mpgermano) [inline orc8r/cloud/Makefile]: Nit: `Essentials` -> `Essential`

**Review by mpgermano — APPROVED** [review]
LGTM

**Pull request merged**